packages/component/src/components/share-menu/use-share-url.ts

```diff
@@ -0,0 +1,51 @@
+import { useAFFiNEI18N } from '@affine/i18n/hooks';
+import { useCallback, useMemo } from 'react';
+
+import { toast } from '../../ui/toast';
+
+export enum UrlType {
+  Public = 'share',
+  UnPublic = 'workspace',
+}
```

Comment:
The word UnPublic sounds strange. I guess you can just use `type UrlType = 'share' | 'workspace'`, without using enums.

Reply:
Made adjustments based on your suggestions